cpp/src/io/utilities/parsing_utils.cuh

```diff
@@ -36,12 +41,47 @@ struct ParseOptions {
   bool doublequote;
   bool dayfirst;
   bool skipblanklines;
-  SerialTrieNode* trueValuesTrie;
-  SerialTrieNode* falseValuesTrie;
-  SerialTrieNode* naValuesTrie;
+  device_span<SerialTrieNode const> trie_true;
+  device_span<SerialTrieNode const> trie_false;
+  device_span<SerialTrieNode const> trie_na;
   bool multi_delimiter;
 };
 
+struct parse_options {
+  char delimiter;
+  char terminator;
+  char quotechar;
+  char decimal;
+  char thousands;
+  char comment;
+  bool keepquotes;
```

Comment:
Should the multi-word members be snake case?
```
keep_quotes
double_quote
day_first
skip_blank_lines
```

Reply:
Probably :shrug:. We can get this cleaned up in a later PR, though. There will be quite a few.

Reply:
I believe these only show up in the files that are already in this PR and you are modifying all their references already.